internal/app/server/repository/migrate.go

```diff
@@ -0,0 +1,31 @@
+package repository
+
+import (
+	"github.com/golang-migrate/migrate/v4"
+
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+)
+
+func ApplyMigrations(filePath string, dsn string) error {
```

Comment:
you can create migrator interface (you'll use migrate.New in main and provide it to the func as an argument) to mock it in tests